src/main/java/Duke.java

```diff
@@ -17,6 +18,11 @@ public static void printGreeting() {
         printDivider();
```

Comment:
I like how you extracted the print divider to a separate function to make your code much neater